packages/gestalt/src/InternalSheet.js

```diff
@@ -74,17 +106,21 @@ export default function InternalSheet({
 
   // Handle onDismiss triggering from ESC keyup event
   useEffect(() => {
-    function handleKeyUp(event: {| keyCode: number |}) {
-      if (event.keyCode === ESCAPE) {
+    function handleKeyDown(event: {| keyCode: number |}) {
```

Comment:
I'm still seeing the `event` Flow type here — is it needed? Shouldn't Flow be able to infer it?

Reply:
DONE